docs/databases/mongo.md

```diff
@@ -20,3 +20,29 @@ databases:
 
 ## Usage
 This module helps opsdroid to persist memory using an MongoDB database.
+
+## Example
+A small change is needed to the [opsdroid's memory feature](../skills/memory.md) example skill in order to use it with the mongo database module:
+
+```python
+from opsdroid.skill import Skill
+from opsdroid.matchers import match_regex
+
+
+class RememberSkill(Skill):
+    @match_regex(r'remember (.*)')
+    async def remember(self, message):
+        remember = message.regex.group(1)
+        await self.opsdroid.memory.put("remember_this", {"value": remember})
```

Comment:
converted `remember` to `{"value": remember}` for mongo example.

Reply:
I don't think this should be a dict. It should just be the value you want to store `remember`.